script.js

```diff
@@ -12,42 +12,33 @@ const country_list = ["Afghanistan","Albania","Algeria","Andorra","Angola","Angu
 	,"Turkey","Turkmenistan","Turks and Caicos","Uganda","Ukraine","United Arab Emirates","United Kingdom","United States","United States Minor Outlying Islands","Uruguay"
   ,"Uzbekistan","Venezuela","Vietnam","Virgin Islands (U.S.)","Yemen","Zambia","Zimbabwe"];
 
-  function getRandomIndex(items) {
+// API
+const api= "https://restcountries.eu/rest/v2/all"
+async function getData(){
+    const res= await fetch(api)
+    const data= await res.json()
+    var print= data.map(country => country.flag)
+    console.log(print)
+    const img = document.querySelector("#img")
+    img.innerHTML = `<img src="${print}" width="150"/>`

```

Comment:
Check your logic here. As my understanding, you are printing the whole `print` array inside the **src** attribute.

So what will show up in the `img.innerHTML` is something similar to `<img src="https://restcountries.eu/data/col.svg,https://restcountries.eu/data/om.svg ..." width="150">`

This is why the image isn't showing in the browser. The **src** attribute should have only one URL per `<img/>`
Is that clear?

Reply:
yes, clear

Reply:
@Rahma-AlWadhahi is correct you are passing an array of SVGs in to the image source .. to pass a single one try `print[0]` this should pass the first image in print array as 0 is the index
so try something like this


 img.innerHTML = `<img src="${print}" width="150"/>`
```suggestion
    img.innerHTML = `<img src="${print[0]}" width="150"/>`

```

Reply:
it didn't work

Reply:
I think I should create an array, but I feel there is a better way to show the flags.

Reply:
I have added another comment. I think that you should fix these two things in order for it to work